app/__init__.py

```diff
@@ -21,7 +21,7 @@ def create_app():
     # CORS Middleware
     app.add_middleware(
         CORSMiddleware,
-        allow_origins=['*'],
+        allow_origins=['http://localhost:3000', 'https://td-uit.no'],
```

Comment:
Then we must either include `https://` and `http://` in the `FRONTEND_URL`, or do it like this:      
`allow_origins=[f"https://{app.config.FRONTEND_URL}", f"http://{app.config.FRONTEND_URL}"]`.

This also is dependant on both of the local frontend_url and the production frontend_url is without the schema

What do you think?

Reply:
Yes of course, imo I think the url  could contain the entire URL. The only drawback is that it could look weird in emails. Maybe a util function getting the entire url may be better🤔